api/notd/manager.py

```diff
@@ -143,9 +143,15 @@ async def process_block(self, blockNumber: int, shouldForce: bool = False) -> No
                 return
         retrievedTokenTransfers = await self.blockProcessor.get_transfers_in_block(blockNumber=blockNumber)
         logging.info(f'Found {len(retrievedTokenTransfers)} token transfers in block #{blockNumber}')
-        await self._create_token_transfers(retrievedTokenTransfers=retrievedTokenTransfers)
-        await asyncio.gather(*[self.update_token_metadata_deferred(registryAddress=retrievedTokenTransfer.registryAddress, tokenId=retrievedTokenTransfer.tokenId) for retrievedTokenTransfer in retrievedTokenTransfers])
-        await asyncio.gather(*[self.update_collection_deferred(address=address) for address in set(retrievedTokenTransfer.registryAddress for retrievedTokenTransfer in retrievedTokenTransfers)])
+        if isinstance(retrievedTokenTransfers,list):
```

Comment:
This looks super wierd I would not have expected it. Can we work to undo these changes and make sure the response from get_transfers is always a list of token tranfers??

Reply:
`INFO:databases:Connected to database postgresql://postgres:********@127.0.0.1:5432/notddb
INFO:root:Found 1 erc1155Batchevents in block #13281282
INFO:root:Found 1 token transfers in block #13281282
INFO:root:Rolling back due to exception: 'list' object has no attribute 'transactionHash'
Traceback (most recent call last):
  File "/home/obafemi/Desktop/nftoftheday/api/local_testing_run.py", line 63, in <module>
    run(_anyio_backend='asyncio')
  File "/home/obafemi/.local/lib/python3.8/site-packages/asyncclick/core.py", line 1159, in __call__
    return anyio.run(self._main, main, args, kwargs, **({"backend":_anyio_backend} if _anyio_backend is not None else {}))
  File "/home/obafemi/.local/lib/python3.8/site-packages/anyio/_core/_eventloop.py", line 56, in run
    return asynclib.run(func, *args, **backend_options)  # type: ignore
  File "/home/obafemi/.local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 230, in run
    return native_run(wrapper(), debug=debug)
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/obafemi/.local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 225, in wrapper
    return await func(*args)
  File "/home/obafemi/.local/lib/python3.8/site-packages/asyncclick/core.py", line 1162, in _main
    return await main(*args, **kwargs)
  File "/home/obafemi/.local/lib/python3.8/site-packages/asyncclick/core.py", line 1083, in main
    rv = await self.invoke(ctx)
  File "/home/obafemi/.local/lib/python3.8/site-packages/asyncclick/core.py", line 1429, in invoke
    return await ctx.invoke(self.callback, **ctx.params)
  File "/home/obafemi/.local/lib/python3.8/site-packages/asyncclick/core.py", line 783, in invoke
    rv = await rv
  File "/home/obafemi/Desktop/nftoftheday/api/local_testing_run.py", line 53, in run
    await notdManager.process_block(blockNumber=blockNumber)
  File "/home/obafemi/Desktop/nftoftheday/api/notd/manager.py", line 152, in process_block
    await self._create_token_transfers(retrievedTokenTransfers=retrievedTokenTransfers)
  File "/home/obafemi/Desktop/nftoftheday/api/notd/manager.py", line 126, in _create_token_transfers
    StringFieldFilter(fieldName=TokenTransfersTable.c.transactionHash.key, eq=retrievedTokenTransfer.transactionHash),
AttributeError: 'list' object has no attribute 'transactionHash'`

Reply:
this is the error i got

Reply:
can you push the code that generates this error please